lib/posix-user/exportsyms.uk

```diff
@@ -1,101 +1,76 @@
-getpid
```

Comment:
I'm not sure how these came to be here (`getpid` and `getppid`) but they are also not defined in any other place.

Reply:
Yes, these do not belong here. They are defined in `posix-process` and exported there as no `exportsyms.uk` means all global symbols are exported.